src/screens/Messages/Conversation/index.tsx

```diff
@@ -56,18 +55,6 @@ function Inner() {
   const convo = useConvo()
   const {_} = useLingui()
 
-  if (
-    convo.status === ConvoStatus.Uninitialized ||
-    convo.status === ConvoStatus.Initializing
-  ) {
-    return (
-      <CenteredView style={a.flex_1} sideBorders>
-        <Header />
-        <ListMaybePlaceholder isLoading />
-      </CenteredView>
-    )
-  }
```

Comment:
This check was intentional, since it would allow us to make a, like, `useSafeConvo` which returns data already in active states. If we remove this we gotta start doing nullish coalescing, like you had to do below. Any chance we can keep this or does it totally mess up loader?